hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveMultitenantNoResolverTest.java

```diff
@@ -150,20 +172,45 @@ public void testOpenSessionWithTenant(TestContext context) {
 		);
 	}
 
+	@Test
+	public void testOpenStatelessSessionWithTenant(TestContext context) {
+		Stage.StatelessSession t1Session = getSessionFactory().openStatelessSession( TENANT_1.name() );
+		test( context, selectCurrentDB( t1Session )
+				.thenAccept( result -> context.assertEquals( TENANT_1.getDbName(), result ) )
+				.whenComplete( (v, e) -> t1Session.close() )
```

Comment:
`session.close()` returns a `CompletionStage` that you are ignoring when using `whenComplete`.
Because this is a test, I think `.thenCompose( v -> t2Session.close() ) )` is good enough.